generators/mvc/templates/packages.config

```diff
@@ -4,14 +4,14 @@
   <package id="DotNetNuke.Instrumentation" version="<%= dnnBuildVersionShort %>" targetFramework="net472" />
   <package id="DotNetNuke.Web" version="<%= dnnBuildVersionShort %>" targetFramework="net472" />
   <package id="DotNetNuke.Web.Client" version="<%= dnnBuildVersionShort %>" targetFramework="net472" />
-  <package id="DotNetNuke.Web.Deprecated" version="<%= dnnBuildVersionShort %>" targetFramework="net472" />
+  <package id="DotNetNuke.Web.Deprecated" version="9.10.2" targetFramework="net472" />
```

Comment:
@ArielBlanco1990 There should be a variable to use for this, instead of hard-coding the version number here. Please follow the pattern that was in place before this update. It previously used dnnBuildVersionShort. If that doesn't work and there's not a dnnBuildVersionLong or something, then we should have it created. Ideally, this update should be done in as few places as possible when we need to do it again in the future.

Reply:
@ArielBlanco1990 Also, it's the wrong version.  If it doesn't exist in 9.11+, then we should remove it entirely.

Reply:
@WillStrohl I followed your instructions and solved this problem. There was no `dnnBuildVersionLong `but there was `dnnBuildVersion `which is what we need. I replaced `<%= dnnBuildVersionShort %>` and `<%= dnnBuildVersion %>` in each case.

I have also removed the reference to `DotNetNuke.Web.Deprecated Version="9.10.2"`